aptos-node/src/lib.rs

```diff
@@ -403,13 +407,6 @@ async fn periodic_telemetry_dump(node_config: NodeConfig, db: DbReaderWriter) {
                 // Build the params from internal prometheus metrics
                 let mut metrics_params: HashMap<String, String> = HashMap::new();
 
-                // Measurement Protocol params must be underscore or alphanumeric
```

Comment:
So here we're getting rid of using prometheus public metrics for telemetry. The only metrics we were gathering via this method were state sync version and number of aptos connections. Both have been replaced with more direct ways of capturing this data, rather than using prometheus metrics. This has the effect of not being weirdly formatted, so we can get rid of the regex hack here that removes the prometheus style of curly braces :)